tests/unit/storage_v2_wal_file.cpp

```diff
@@ -621,3 +635,80 @@ TEST_P(WalFileTest, PartialData) {
   ASSERT_EQ(pos, infos.size() - 2);
   AssertWalInfoEqual(infos[infos.size() - 1].second, memgraph::storage::durability::ReadWalInfo(current_file));
 }
+
+std::string_view StorageModeToString(memgraph::storage::StorageMode storage_mode) {
```

Comment:
This code is duplication. It seems to me like a good idea to extract it and put it under storage_test_utils or some file similar to this.